packages/wrangler/src/__tests__/helpers/mock-account-id.ts

```diff
@@ -9,14 +10,15 @@ const ORIGINAL_CLOUDFLARE_ACCOUNT_ID = process.env.CLOUDFLARE_ACCOUNT_ID;
  * Note that you can remove any API token from the environment by setting the value to `null`.
  * This is useful if a higher `describe()` block has already called `mockApiToken()`.
  */
+
 export function mockApiToken({
-  apiToken = "some-api-token",
-}: { apiToken?: string | null } = {}) {
+  apiToken = { token: "some-api-token", type: "OAuth" },
+}: { apiToken?: TokenWithType } = {}) {
   beforeEach(() => {
-    if (apiToken === null) {
+    if (apiToken?.token === null) {
```

Comment:
`apiToken` could be `undefined` which would result in `apiToken?.token === null` being `false`. Is that what you want?

Reply:
Oh, it really is! Nice.